link-grammar/print/print-util.c

```diff
@@ -50,8 +63,14 @@ size_t utf8_strwidth(const char *s)
 
 /* ============================================================= */
 
-/* Note: As in the rest of the LG library, we assume here C99 compliance. */
-void vappend_string(dyn_str * string, const char *fmt, va_list args)
+/**
+ * Append to a dynamic string with vprintf-like formatting.
+ * @return The number of appended characters.
```

Comment:
Number of appended bytes, or number of appended glyphs?

Reply:
It is the number of bytes.
However, the vsnprintf() Linux manual use the term "characters" for that.
The BSD manual also uses the term "characters".

---
RETURN VALUE
Upon successful return, these functions return the number of characters printed (excluding the null byte used to end  output to strings).

---
However, I neglected to mention that a negative value is returned in case of error.
So I will add that and change it to bytes too.